lib/main/atom/tooltipManager.ts

```diff
@@ -15,10 +15,19 @@ import escape = require("escape-html")
 
 export function getFromShadowDom(element: JQuery, selector: string): JQuery {
   var el = element[0]
-  var found = (<any>el).rootElement.querySelectorAll(selector)
+  var found = (<any>el).querySelectorAll(selector)
```

Comment:
Frankly a lot of code here begs for a rewrite. Why this particular problem is singled out, considering the code base is littered with stuff like this, eludes me. Anyway, this repo is half-dead, and I neither want nor can maintain it (have my hands full as it is), so unless someone with actual commit rights outright *refuses* to both merge and fix issues that prompted this PR otherwise, I'm not inclined to do the clean-up job. Sorry if I sound a bit harsh, today was a long day, and I might be a bit on edge.